plankton/devices/linkam_t95/interfaces/stream_interface.py

```diff
@@ -21,6 +21,19 @@
 
 
 class LinkamT95StreamInterface(StreamAdapter):
+    """
+    Linkam T95 TCP stream interface
+
+    This is the interface of a simulated Linkam T95 device. The device listens on a configured
+    host:port-combination, one option to connect to it is via telnet:
+
+        $ telnet host port
+
+    Once connected, it's possible to send the specified commands, described in the dynamically
+    generated documentation. Information about host, port and line terminators in the concrete
+    device instance are also generated dynamically.
```

Comment:
Very cool! I don't think I've seen that before (or I've just completely forgotten).

Yes, I think shipping something like that in scripts would be perfect.

If other protocols need it, we could extend it (or protocol developers could provide their own interface tool).

A potential alternative to consider would be to integrate both this sort of functionality and the current control client into a common entry point with subcommands (similar to docker or git):

```
$ plankton start chopper -r localhost:2020
$ plankton control localhost:2020 device
$ plankton connect localhost:9999
```

But we should probably create a separate issue for discussion on that.